psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -346,67 +346,51 @@ void DFMP2::form_singles() {
     SharedVector eps_aocc_b = epsilon_b_subset("SO", "ACTIVE_OCC");
     SharedVector eps_avir_b = epsilon_b_subset("SO", "ACTIVE_VIR");
 
-    auto Fia_a = std::make_shared<Matrix>("Fia a", Caocc_a->colspi(), Cavir_a->colspi());
-    auto Fia_b = std::make_shared<Matrix>("Fia b", Caocc_b->colspi(), Cavir_b->colspi());
+    auto Fia_a = linalg::triplet(Caocc_a, Fa_, Cavir_a, true, false, false);
+    auto Fia_b = linalg::triplet(Caocc_b, Fb_, Cavir_b, true, false, false);
```

Comment:
smart `triplet`. guess the dfmp2 module was from days before the invention of `doublet` and `triplet`.